.golangci.yml

```diff
@@ -23,6 +23,7 @@ linters:
     - structcheck
     - unconvert
     - varcheck
+    - goimports
```

Comment:
nit: the others are sorted alphabetically

Reply:
Great catch! TY